code/game/mecha/mecha_actions.dm

```diff
@@ -217,9 +217,9 @@
 			new_damtype = "brute"
 			chassis.occupant_message("Your exosuit's hands form into fists.")
 		if("brute")
-			new_damtype = "fire"
+			new_damtype = FIRE
```

Comment:
Both of these two should stay as "fire" for consistency with the rest of the proc. (As much consistency as you can get with mech code, anyway.)
https://github.com/ParadiseSS13/Paradise/blob/1465a7de14c5090c7dede80847f63e7f3dc8935e/code/game/mecha/mecha_actions.dm#L211-L228
```suggestion
			new_damtype = "fire"
```